Create modify-compressed-data.md (#2389)
timescale/docs: api/compression.md

Compressed chunks have the following limitations:

*   `ROW LEVEL SECURITY` is not supported.
*   Creation of unique constraints on compressed chunks is not supported

>    *   Its possible to add them by disabling compression and re-enabling after constraint creation.

Reviewer: ```suggestion
   *   You can add them by disabling compression and re-enabling after constraint creation.
```

| 1.5 - 2.0 | Data and schema modifications are not supported. |
| 2.1 - 2.2 | Schema may be modified on compressed hypertables. Data modification not supported. |
| 2.3 | Schema modifications and basic insert of new data are allowed. Deleting, updating and some advanced insert statements are not supported. |

> | 2.11 | Deleting, updating and advanced insert statements are supported. |

Reviewer: All tables should be unspaced, but it's not a deal breaker.
